# 单个检测结果消息
string class_name
int32 class_id
float32 confidence
geometry_msgs/Point center
float32 width
float32 height
geometry_msgs/Polygon bbox_points